### ISSUE TITLE
Taking powerups displays their respective chat messages too many times

### ISSUE THREAD

**Issue by Ducks9920** [zapzombies/zap-community#21]
**Describe the bug**
If you take a powerup, you will see chat messages such as "Duckz__ activated Insta Kill" multiple times instead of only once.

**Where did the bug occur**
map_nt

**To Reproduce**
1. Take a powerup
2. Look at chat

**Expected behavior**
The chat message is supposed to show up only once.

**Screenshots**
look at chat, it's crazy
![2021-09-19_11 33 58](https://user-images.githubusercontent.com/91029590/133940173-26b2e6de-fb2f-4118-a651-0485584a8c58.png)

**Additional context**
no

**Comment by nekorii04** [comment]
nice 4k monitor bro
steank already said he will fix thi
![glowing-z-letter-z](https://user-images.githubusercontent.com/90960152/133940215-91b14708-d510-4da6-8fdf-bbaab943cd7c.gif)
s

**Comment by Steanky** [comment]
I labelled "low priority" but i'm still going to try and fix this right away lol

**Comment by nekorii04** [comment]
GG

**Comment by thamid-gamer** [comment]
I'm going to not sharpen one of my pencils that I bring to school even though it's broken if the code I just wrote doesn't fix this

**Comment by tachibanayui** [comment]
There are no commit related to fixing this?

**Comment by thamid-gamer** [comment]
I just tested and it seems fine
zapzombies/zap-community@ebc6bdfe6334d2d53f63c96418632f710b6a89d7
